### PR TITLE
Allow for namelocking without their exact current name

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

File: server/chat-commands.js [hunk 1]
```diff
@@ -2582,10 +2582,11 @@ const commands = {
 	],
 
 	nl: 'namelock',
-	namelock(target, room, user) {
+	forcenamelock: 'namelock',
+	namelock(target, room, user, connection, cmd) {
 		if (!target) return this.parse('/help namelock');
 
-		let reason = this.splitTarget(target, true);
+		let reason = this.splitTarget(target, cmd !== 'forcenamelock');
```
Comment (Zarel) [inline server/chat-commands.js]: Could you make a follow-up to actually send a reminder to use `/forcenamelock` if someone's renamed?

**Comment by Zarel** [comment]
Apparently I forgot to merge this. I said "could you make a follow-up" because I was going to merge it as-is.

**Comment by Zarel** [comment]
Anyway, thanks! This is exactly what I was looking for!

**Comment by scheibo** [comment]
> Apparently I forgot to merge this. I said "could you make a follow-up" because I was going to merge it as-is.

This is what I figured.

**Pull request merged**